### PR TITLE
energy revolver & laser changes

### PR DESCRIPTION
**Pull request by W0LF-101**
Energy Revolver Changes:
No longer selfcharges 
Cell charge reduced from 1500 to 1000
Takes more power to fire lethal and taser shots (10% & 20% vs 5% and 13%) 

SC-1 Laser Gun Changes:
Takes less power to fire lethal shots (from 6% to 5%)

### CODE REVIEW

**Comment by W0LF-101** [comment]
Please testmerge this alongside PR #98 Energy Sec as this change buffs the SC-1s to encourage picking those when the nonlethal ability of the SC-2s is not desired.

File: code/modules/projectiles/ammunition/energy/laser.dm [hunk 1]
```diff
@@ -2,7 +2,10 @@
 	projectile_type = /obj/projectile/beam/laser
 	e_cost = 83
 	select_name = "kill"
-
+/obj/item/ammo_casing/energy/laser/revolver
+	projectile_type = /obj/projectile/beam/laser
```
Comment (Cyprex) [inline code/modules/projectiles/ammunition/energy/laser.dm:6]:
```suggestion

```

File: code/modules/projectiles/ammunition/energy/laser.dm [hunk 2]
```diff
@@ -2,7 +2,10 @@
...
+	e_cost = 100
+	select_name = "kill"
```
Comment (Cyprex) [inline code/modules/projectiles/ammunition/energy/laser.dm:8]:
```suggestion
```

Comment (Cyprex) [reply code/modules/projectiles/ammunition/energy/laser.dm:8]: Removing these takes the values from `/obj/item/ammo_casing/energy/laser` instead, which is what you want here.
Also this ought to be modularised, I can handle that if you wish.

**Review by Cyprex — CHANGES_REQUESTED** [review]
Looks good otherwise

File: modular_skyrat/modules/blueshield/code/special.dm [hunk 3]
```diff
@@ -14,11 +14,11 @@
 	obj_flags = UNIQUE_RENAME
 	cell_type = /obj/item/stock_parts/cell/blueshield
 	pin = /obj/item/firing_pin/implant/mindshield
-	selfcharge = TRUE
+	selfcharge = FALSE
 
 /obj/item/stock_parts/cell/blueshield
 	name = "internal revolver power cell"
-	maxcharge = 1500
+	maxcharge = 1000
```
Comment (projectkepler-ru) [inline modular_skyrat/modules/blueshield/code/special.dm:21]: keep the max charge at 1500

**Review by projectkepler-ru — CHANGES_REQUESTED** [review]

**Review by projectkepler-ru — CHANGES_REQUESTED** [review]
Scratch that, change it to 800 and reenable self charging, I'm reintroducing it as a blueshield sidearm then this is good to merge

**Review by projectkepler-ru — CHANGES_REQUESTED** [review]

**Pull request merged**